contracts/multisig/src/contract.rs

```diff
@@ -166,11 +179,16 @@ impl Multisig {
         }
 
         let proposal_id = increment_last_proposal_id(&env);
+        let creation_timestamp = env.ledger().timestamp();
+        let expiration_timestamp = creation_timestamp + deadline.unwrap_or(SEVEN_DAYS_DEADLINE);
+
```

Comment:
Checks are missing like mentioned above, and here if `deadline` is not present just don't modify the value.

Reply:
I added the same check as in `create_transaction_proposal`